package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "jaeger-client-javascript",
```

Comment:
Would it make sense to have a shorter name, like `jaegerjs`? Not sure what the NPM conventions are around that.

Reply:
I suggest you consider reserving the `@jaeger` npm scope, if that's still available.

`@jaeger/client` has a nice ring to it, and scopes have a few other benefits, like making it easy for users to identify packages "officially sanctioned" by the Jaeger org.

Reply:
Good idea!  In fact, we already have it, `@jaegertracing`, so far with only one package https://www.npmjs.com/package/@jaegertracing/plexus